docker/clightning/Dockerfile

```diff
@@ -99,19 +99,29 @@ RUN git clone --recursive --branch=v${CLN_VERSION} https://github.com/ElementsPr
 # Original line: ARG DEVELOPER=0
 ARG DEVELOPER=1
 ####################
+
+ENV PYTHON_VERSION=3
+
+#################### Polar Modification
+# Use fixed version of mistune, because the latest version breaks the mrkd package
+# https://github.com/ElementsProject/lightning/issues/4973
+RUN pip3 install mrkd mistune==0.8.4
```

Comment:
There's been a more recent update in the `lightning` repo to fix the `mrkd` package 
```
RUN apt-get install -y --no-install-recommends python3-dev
RUN pip3 install -U pip && pip3 install -r requirements.lock
```

See https://github.com/ElementsProject/lightning/pull/4969